src/plugins/cordova-plugin-globalization/sim-host-panels.html

```diff
@@ -1,7 +1,22 @@
 <!-- Copyright (c) Microsoft Corporation. All rights reserved. -->
 
 <cordova-panel id="globalization" caption="Globalization" spoken-text="Globalization Panel" data-loc-id="5ffc4b1e">
-    <cordova-combo id="locale-list" label="Locale" data-loc-id="9e831385"></cordova-combo>
+    <cordova-combo id="locale-list" label="Locale" data-loc-id="9e831385">
+    	<option value="en-US" data-loc-id="0ebb8953">English</option>
```

Comment:
Cool. I have to run the tool to test my change, or a localized version will not have the change. I'd not change it back and forth this time. So let's leave it for now.